File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task59_GEMINI_gemini-1.5-pro-001.java
public class Accumul {

    public static String accum(String s) {
        StringBuilder sb = new StringBuilder();
        int i = 0;
        for(char c : s.toCharArray()) {
            if (i > 0) sb.append('-');
            sb.append(Character.toUpperCase(c));
            for (int j = 0; j < i; j++) sb.append(Character.toLowerCase(c));
            i++;
        }
        return sb.toString();
  }
    public static void main(String[] args) {
        System.out.println(accum("abcd"));
        System.out.println(accum("RqaEzty"));
        System.out.println(accum("cwAt"));
    }
}

File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task59_GEMINI_gemini-1.5-pro-001.cpp
#include <iostream>
#include <string>
#include <algorithm>

std::string accum(const std::string& s) {
  std::string result;
  for (size_t i = 0; i < s.length(); ++i) {
    if (i > 0) result += "-";
    result += std::toupper(s[i]);
    result += std::string(i, std::tolower(s[i]));
  }
  return result;
}

int main() {
  std::cout << accum("abcd") << std::endl;
  std::cout << accum("RqaEzty") << std::endl;
  std::cout << accum("cwAt") << std::endl;
  return 0;
}